### PR TITLE
Update Weld jars

### PR DESCRIPTION
**Pull request by tevans78**

### CODE REVIEW

File: dev/com.ibm.ws.org.jboss.weld3/bnd.overrides [hunk 1]
```diff
@@ -12,7 +12,9 @@ Specification-Title: JSR-365 Contexts and Dependency Injection for Java
 Specification-Version: 2.0
 DynamicImport-Package: *
 Export-Package: org.jboss.*;version="3.1.1.Final", \
+ !org.jboss.environment.se.*, \
```
Comment (Azquelt) [inline dev/com.ibm.ws.org.jboss.weld3/bnd.overrides]: The parameters to `Export-Package` are processed in order. To get the behaviour you're looking for, you need to put the exclusion first.

For reference see https://bnd.bndtools.org/heads/private_package.html

As a side note, the current bundle manifest ends up exporting `org.jboss.weldx.transaction` twice, once at v3.1.1.Final and once at 1.0.16. The package does exist within the final bundle but contains no classes. I can't find this package in any of the source jars.

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
PB failures are unrelated to this. Merging.

**Pull request merged**